update balance after p to p transfer

aeddaqqa opened the pull request.

aeddaqqa committed:
update balance after p to p transfer

Ayagoumi commented on src/views/wallet/Transfer.vue:
```diff
@@ -139,14 +139,22 @@
+                            <div v-if="formType === 'P' && isMultiSig">
+                                <p style="color: var(--success)">
+                                    <fa icon="check-circle"></fa>
+                                    Pending transaction created
```
Add translation here

jamesckhsiao committed:
fix(e2e): modify e2e tests to adopt to the latest ui

- add cypress test attribute `data-cy` to the test target
- modify related tests to use `data-cy` attribute instead of querying dom by hierarchy selectors

aeddaqqa committed:
adjust the style of memo/toAddress/anount for a tx in pending state

aeddaqqa committed:
fix start new transaction in multisig transfer

aeddaqqa committed:
Merge branch 'suite' into suite-update-balance

aeddaqqa committed:
fixes

aeddaqqa committed:
Merge branch 'suite' into suite-update-balance

aeddaqqa committed:
Merge branch 'suite' into suite-update-balance
